packages/app/server/services/vite-service.ts

```
@@ -0,0 +1,70 @@
+
+	static getInstanceEffectually(): Effect.Effect<ViteServiceSingleton> {
+		return Effect.promise(() => ViteServiceSingleton.getInstance())
+	}
+
+	/**
+	 * Asynchronously disposes of the resource associated with the object.
+	 * This method is invoked during object cleanup when the `Symbol.asyncDispose` is triggered.
+	 *
+	 * @return {Promise<void>} A promise that resolves once the resource has been successfully disposed of.
+	 */
+	public async [Symbol.asyncDispose](): Promise<void> {
```

Biome (1.9.4)

[error] The public modifier is disallowed.

Remove the accessibility modifier.

(lint/nursery/useConsistentMemberAccessibility)